include/colors.h

```diff
@@ -16,14 +16,13 @@ struct Color {
   Color(std::string);
   std::string eps();
 
-  bool operator == (const Color& rhs) const
+  bool operator==(const Color &rhs) const
   {
-      return (r == rhs.r && g == rhs.g && b == rhs.b);
+    return (r == rhs.r && g == rhs.g && b == rhs.b);
   }
 };
 
-static std::unordered_map<std::string, std::string> html_colors =
-{
+static std::unordered_map<std::string, std::string> html_colors = {
```

Comment:
I changed it such that it is a local static variable. I will shortly set up clang-tidy on my own computer so that I can check for suggestions myself before pushing code. However, for the moment being it may be faster if you were to test whether this still causes clang-tidy to complain.